interface/main/messages/print_postcards.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
```

Comment:
>  The ‘Appt Reminder' items 'Calendar' and 'Flow Board' should open up in a separate tab when click these (this will avoid losing menu and also avoiding duplicate tags of same screen).

OK so the last commit has a javascript solution for this.